tutorials/katacoda/getting-started/src/tekton-katacoda/tasks/build.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: build-test-app
+spec:
+  inputs:
```

Comment:
should be `resources.inputs` for v1beta1
and `inputs.params` should just be `params`